src/server/handlers/firestore/transactions.ts

```diff
@@ -157,7 +158,7 @@ async function handleAuthorization(server: StateServer, transaction: Transaction
 
   // TD - LD eww so much messy casting going on
   // @ts-ignore - todo fix me!
-  const requestBody: PutThirdpartyRequestsTransactionsAuthorizationsRequest = {
+  const requestBody: tpAPI.Schemas.ThirdpartyRequestsTransactionsIDAuthorizationsPutResponsequest = {
```

Comment:
typo?

Reply:
Haha yeah I guess the `ts-ignore` should be removed and we would see a built time error.